xml/dns_versionbind.xml

```diff
@@ -0,0 +1,685 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  This fingerprint file matches the text string response from a DNS
+  version.bind request.
+
+  For example, the string 'dnsmasq-2.76-1-ubnt2' emitted by the command below:
+
+  $ nslookup -type=txt -class=chaos VERSION.BIND <dns_server> | grep VERSION.BIND | cut -d\" -f2
+  dnsmasq-2.76-1-ubnt2
+
+-->
+<fingerprints matches="dns_versionbind" protocol="dns" database_type="service" preference="0.750">
```

Comment:
There is no standard on this necessarily, but `dns.versionbind` is probably more consistent for `matches`